packages/console/src/icons/Copy.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { SVGProps } from 'react';
+
+const Copy = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
Maybe we can share the `Copy` icon with the `CopyToClipboard` component.

Reply:
Not sure if this will break UI, from svg, they are not the same.